### PR TITLE
Remove old post-process shader pipeline

### PR DESCRIPTION
**Pull request by bejado**
This removes the old concept of post-process shaders from Filament, including:
- The `-m postprocess` flag in matc
- The concept of "post process versions"
- `PostProcessMaterialBuilder` / `PostProcessMaterialCompiler`
- Old post-process code generation
- Old post-process shaders

### CODE REVIEW

File: filament/src/MaterialParser.h [hunk 1]
```diff
@@ -50,11 +50,9 @@ class MaterialParser {
 
     bool parse() noexcept;
     bool isShadingMaterial() const noexcept;
```
Comment (romainguy) [inline filament/src/MaterialParser.h]: We probably don't need this one anymore then? We should also have a getDomain() if we don't already

File: tools/matinfo/src/main.cpp [hunk 2]
```diff
@@ -376,8 +376,7 @@ static bool parseChunks(Config config, void* data, size_t size) {
 
         if (config.printGLSL) {
             ShaderExtractor parser(Backend::OPENGL, data, size);
-            if (!parser.parse() ||
-                    (!parser.isShadingMaterial() && !parser.isPostProcessMaterial())) {
+            if (!parser.parse() || !parser.isShadingMaterial()) {
```
Comment (romainguy) [inline tools/matinfo/src/main.cpp]: isShadingMaterial is useless now right?

File: filament/src/MaterialParser.h (hunk 1 above)
Comment (bejado) [reply filament/src/MaterialParser.h]: Good point, yeah it can probably be removed. I'll add a `getDomain` too.

Comment (bejado) [reply filament/src/MaterialParser.h]: Oh, just realized I already added a getter: `getMaterialDomain` on line 73.

**Pull request merged**